server/routes/visit.ts

```diff
@@ -85,7 +84,8 @@ export default function routes({
     const visitSessionData: VisitSessionData = {
       allowOverBooking: false,
       prisoner: {
-        name: properCaseFullName(`${prisoner.lastName}, ${prisoner.firstName}`),
+        firstName: prisoner.firstName,
+        lastName: prisoner.lastName,
```

Comment:
```suggestion
        firstName: convertToTitleCase(prisoner.firstName),
        lastName: convertToTitleCase(prisoner.lastName),
```
The API returns prisoner name as **upper case** so it needs to be formatted.